index.html

```diff
@@ -48,5 +48,162 @@ <h1 class="main-section__introduction">I'm Mariana Glad to see you</h1>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <div class="project-tech">
+            <span>html</span>
+            <span>css</span>
+            <span>javascript</span>
+          </div>
+          <div class="project-button">
+            <a href="">See Project</a>
+          </div>
+        </div>
+      </div>
+
+      <div class="project-items">
+        <img src="images/portfolio2.png" alt="Project photo" />
+        <div class="project-info">
+          <h2>Multi-Post Stories</h2>
+          <div class="project-details">
+            <span class="project-detail">CANOPY</span>
+            <ul class="project-stack">
+              <li>Back End Dev</li>
+              <li>2015</li>
+            </ul>
+          </div>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <div class="project-tech">
+            <span>html</span>
+            <span>css</span>
+            <span>javascript</span>
+          </div>
+          <div class="project-button">
+            <a href="">See Project</a>
+          </div>
+        </div>
+      </div>
+      <div class="project-items">
+        <img src="images/portfolio1.png" alt="Project photo" />
+        <div class="project-info">
+          <h2>Tonic</h2>
+          <div class="project-details">
+            <span class="project-detail">CANOPY</span>
+            <ul class="project-stack">
+              <li>Back End Dev</li>
+              <li>2015</li>
+            </ul>
+          </div>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <div class="project-tech">
+            <span>html</span>
+            <span>css</span>
+            <span>javascript</span>
+          </div>
+          <div class="project-button">
+            <a href="">See Project</a>
+          </div>
+        </div>
+      </div>
+      <div class="project-items">
+        <img src="images/portfolio1.png" alt="Project photo" />
+        <div class="project-info">
+          <h2>Multi-Post Stories</h2>
+          <div class="project-details">
+            <span class="project-detail">CANOPY</span>
+            <ul class="project-stack">
+              <li>Back End Dev</li>
+              <li>2015</li>
+            </ul>
+          </div>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <div class="project-tech">
+            <span>html</span>
+            <span>css</span>
+            <span>javascript</span>
+          </div>
+          <div class="project-button">
+            <a href="">See Project</a>
+          </div>
+        </div>
+      </div>
+    </section>
+
+    <section class="about-section">
```

Comment:
- [ ] kindly add the missing border-radius on the top right corner of the about section to match figma design, since you asked for help to do it here are my suggestions :

1. change the body color to `#E5E5E5` 
2. change the background color of the `about-section` to white
3. and then add `border-top-right-radius` to the  `about-section` 


**_summary :_**
```CSS
body { 
/* update bg color for body */
background: #E5E5E5;
}

.about-section {
/* update bg color for about section and add border radius*/
border-top-right-radius: 100px;
background: white;
}
```

![image](https://user-images.githubusercontent.com/89282221/192093446-de89d6ab-75d7-4ebb-8d5a-a337012a37ea.png)

Reply:
Hi @ichala I have done what you wrote but body is not changed